templates/test/ci/cluster-template-prow-windows.yaml

```diff
@@ -298,6 +298,10 @@ spec:
           New-HnsNetwork -Type Overlay -AddressPrefix "192.168.255.0/30" -Gateway "192.168.255.1" -Name "External" -AdapterName "Ethernet 2" -SubnetPolicies @(@{Type = "VSID"; VSID = 9999; })
         path: C:/create-external-network.ps1
         permissions: "0744"
+      - content: |
+          New-Item -ItemType Directory -Force -Path C:\tmp\
+        path: C:/create-temp-folder.ps1
```

Comment:
what is this temp folder useful for?

Reply:
It is required for the end to end test. It is assume to be already created on the machines for several of the tests. I swear I left a comment here but it looks like it might have been dropped during a rebase